docs/sources/dashboards/build-dashboards/create-dashboard/index.md

```diff
@@ -29,11 +29,21 @@ Dashboards and panels allow you to show your data in visual form. Each panel nee
 
    ![Empty dashboard state](/media/docs/grafana/dashboards/empty-dashboard-9.5.png)
 
-1. In the first line of the **Query** tab, click the dropdown list and select a data source.
-1. Write or construct a query in the query language of your data source.
+1. In the modal that opens, do one of the following:
+
+   - Select one of your existing data sources.
+   - Select one of the Grafana's [built-in special data sources]({{< relref "../../../datasources/#special-data-sources" >}}).
+   - Click **Configure a new data source** to set up a new one.
```

Comment:
There is a case where the user might not have admin permissions. Then, the button is disabled and they cannot add permissions.

Do you think it is worth mentioning? It should be a tooltip appearing saying you don't have admin permissions, but maybe worth mentioning here.

Reply:
Good call. What about this?

```suggestion
   - Click **Configure a new data source** to set up a new one (Admin users only).
```